vuepress/docs/v6.3.2/docs/releases/README.md

```diff
@@ -100,5 +100,8 @@
   - There is  a known issue with the current nginx ingress that can be worked around by using the `singleHost` option for the configuration of your Entando Applications and binding TLS to that single host path
     - The issue will be solved in the next minor or patch release (6.4.0 or 6.3.3)
 
+### Apple M1
```

Comment:
This reads well, nice job. Minor suggestion, do we want the known issues to be alpha-sorted - Apple M1, then Entando Component Generator, then GKE?

Reply:
@nshaw Any opinion on omitting "issue" as @jyunmitch suggested?

Reply:
I like @jyunmitch's suggestion, bit more concise, but both read fine to me.